server/src/inmantals/server.py

```diff
@@ -629,7 +644,7 @@ def convert_location(self, location: Location):
     def get_definition(self, target: AnchorTarget) -> str:
         file_path = target.location.file
         start_line = target.location.lnr - 1
-        with open(file_path, "r") as f:
+        with open(file_path, "r", encoding="utf8") as f:
```

Comment:
Fixes a bug when hovering e.g. over `std::none` in iso4. When encoding is not specified, locale.getpreferredencoding(False) is called to get the current locale encoding and it resolves to `ANSI_X3.4-1968` on my system and it throws this error:
`  Message: 'ascii' codec can't decode byte 0xc2 in position 1113: ordinal not in range(128)`